mapper-runtime/src/main/java/com/datastax/oss/driver/api/mapper/MapperContext.java

```diff
@@ -45,6 +45,13 @@
   @Nullable
   CqlIdentifier getTableId();
 
+  /**
+   * If this context belongs to a DAO that was built with a execution-profile-parameterized mapper
+   * method, the value of that parameter. Otherwise null.
+   */
+  @Nullable
+  String getExecutionProfileName();
```

Comment:
I guess I'm questioning how important it is to maintain backwards compatibility with the MapperContext interface... I could stick this into a custom state, my issue with that is that then the support for the @Daoprofile annotation is implicit instead of explicit.

Reply:
I don't think it needs to be implicit. Please see how it is handled for `schema-validation`, there is an explicit setting: 
```
public MapperBuilder<MapperT> withSchemaValidationEnabled(boolean enableSchemaValidation) {
```

Reply:
I discussed this with Olivier, he's okay with my current implementation.